src/stacked.rs

```diff
@@ -4,16 +4,15 @@ use probe_rs::{Core, MemoryInterface};
 
 /// Registers stacked on exception entry.
 #[derive(Debug)]
-pub struct Stacked {
+pub(crate) struct Stacked {
     r0: u32,
     r1: u32,
     r2: u32,
     r3: u32,
     r12: u32,
```

Comment:
Do we have to keep reading these?

Reply:
actually, we don't! new commit avoids reading these first 5 registers